### ISSUE TITLE
Add URL Annotation

### ISSUE THREAD

**Issue by philkra** [elastic/elastic-client-generator#277]
## 🚀 Feature Proposal

Annotation that associates an URL to a property and/or type, that is then reflected in the spec.

## Motivation

Additional meta data to improve e.g. developer experience to link to a resource for more context.

## Example

in TypeScript:
```
/** @url https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#byte-units */
type ByteSize = string
```

in the spec:
```
{
    "name": "data.input_bytes",
    "required": false,
    "type": {
    "kind": "instance_of",
    "type": {
        "name": "ByteSize",
        "namespace": "internal",
        "url": "https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#byte-units"
    }
}
```

**Comment by delvedor** [comment]
This has already been done in https://github.com/elastic/elastic-client-generator/pull/260 :)
The js doc tag to be used is `@doc_url`. It looks like the docs needs to be updated.

**Comment by delvedor** [comment]
Closed in #324.
